torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -9,16 +9,19 @@
 from itertools import chain
 from typing import Any, Callable, Dict, List, Optional, Type
 
+import logging
 import torch
 import torch.distributed as dist
 from torch.optim import Optimizer
-import logging
 
 __all__ = ["ZeroRedundancyOptimizer"]
 
 
 # Credits:  classy_vision/generic/distributed_util.py
-def _recursive_copy_to_device(value: Any, non_blocking: bool, device: torch.device) -> Any:
+def _recursive_copy_to_device(
```

Comment:
(no need to change this PR, just a heads-up) there is another `recursive_to` in DDP. It might be helpful to consolidate these two into one common utility function. 

https://github.com/pytorch/pytorch/blob/4887c6e401b73c6b024a344076899fd8cdf4a26b/torch/nn/parallel/distributed.py#L907